src/pilot/raid.patch

```diff
@@ -51,7 +52,23 @@
  def create_virtual_disk(node, raid_controller, physical_disks, raid_level,
                          size_mb, disk_name=None, span_length=None,
                          span_depth=None):
-@@ -202,7 +232,108 @@
+@@ -159,7 +190,14 @@
+     drac_job.validate_job_queue(node)
+ 
+     client = drac_common.get_drac_client(node)
+-
++    LOG.info("Properties before creating VD")
```

Comment:
This looks like debugging code.  This code isn't present upstream.

Reply:
This patch also includes the changes for the drive size changing during conversion.
I'll go ahead and merge this, but I think this patch should be revisited when the backports are complete.